src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -12464,14 +12464,14 @@ def test_aks_approuting_enable_disable(
         delete_cmd = "aks delete --resource-group={resource_group} --name={aks_name} --yes --no-wait"
         self.cmd(delete_cmd, checks=[self.is_empty()])
 
-    @AllowLargeResponse()
+    @AllowLargeResponse(8192)
     @AKSCustomResourceGroupPreparer(
         random_name_length=17,
         name_prefix="clitest",
         location="eastus",
         preserve_default_location=True,
     )
-    def test_aks_approuting_enable_with_keyvault_secrets_provider_addon(
```

Comment:
Please remove the old recording file corresponding to this test case with name `test_aks_approuting_enable_with_keyvault_secrets_provider_addon.yaml`

Reply:
removed